src/scss/tools/_typography.scss

```diff
@@ -89,11 +89,18 @@
 
 @mixin input-label {
   font-size: 14px;
-  line-height: 20px;
   font-weight: bold;
   color: $level-3-body-and-headings;
 }
 
+@mixin table-headers {
+  font-size: 14px;
+  line-height: 20px;
+  font-weight: bold;
+  text-transform: unset;
```

Comment:
I know I suggested doing some transformation (from previous uppercase to Titlecase) maybe, shall we deal with this at the component level if any?